x-pack/packages/kbn-infra-forge/src/run.ts

```diff
@@ -37,14 +37,28 @@ const createEvents = (size: number, timestamp: Moment) =>
     })
     .flat();
 
-function indexHistory(lookback: string, queue: any, logger: ToolingLog, nextTimestamp?: Moment) {
+function indexHistory(
+  lookback: string,
+  queue: any,
+  logger: ToolingLog,
+  eventsPerCycle?: number,
```

Comment:
I added this in an attempt to generate more data, but I need to reconsider, since I am not using actual data in my basic tests.